internal/cmd/stack/environment.go

```diff
@@ -30,6 +32,46 @@ type ConfigInput struct {
 	WriteOnly graphql.Boolean `json:"writeOnly"`
 }
 
+type configElement struct {
+	ID        string     `graphql:"id" json:"id,omitempty"`
+	Checksum  string     `graphql:"checksum" json:"checksum,omitempty"`
+	CreatedAt int64      `graphql:"createdAt" json:"createdAt,omitempty"`
+	Runtime   bool       `graphql:"runtime" json:"runtime,omitempty"`
+	Type      ConfigType `graphql:"type" json:"type,omitempty"`
+	Value     *string    `graphql:"value" json:"value,omitempty"`
+	WriteOnly bool       `graphql:"writeOnly" json:"writeOnly,omitempty"`
+	FileMode  *string    `graphql:"fileMode" json:"fileMode,omitempty"`
+}
+
+type listEnvElementOutput struct {
+	Name     string  `json:"name"`
+	Type     string  `json:"type"`
+	Value    *string `json:"value"`
+	FileMode *string `json:"fileMode"`
+	// Context specifies the name of the context.
+	Context *string `json:"context"`
+	// Runtime is not printed, it's just used to determine output formatting.
+	Runtime bool
+	// WriteOnly is not printed, it's just used to determine output formatting.
+	WriteOnly bool
```

Comment:
Can you add json struct tags to `Runtime` and `WriteOnly`. They aren't displayed in the output, but they're used in the JSON output. Otherwise we end up with a mixture of cases.

Reply:
Sure, I've updated the code.